scripts/compare.py

```diff
@@ -125,13 +126,23 @@ def compare(old_polygons, new_polygons):
 		print('Processing SVG child elements:', new_element_id)
 
 		overlaps = {}
+		nearest_markers = {}
 
-		polygon = new_polygon
 		for old_element_id, old_polygon in old_polygons.items():
 
+			#FIXME: We don't need the actual intersection, but just if it's too close.
+			#       So if we the bounding boxes aren't close enough to be a problem, then we already don't care.
+			#FIXME: Retrieving bounds is slow, so we should cache them?
 			if not new_polygon.intersects(old_polygon):
-				#FIXME: Get distance to check if someone might have left an accidental gap, or moved too close
-				pass
+				# Get distance to check if someone might have left an accidental gap, or moved too close.
+				#FIXME: This step appears to be slow, so we should make this optional?
+				if False:
```

Comment:
Disabled because it's way too slow.
I assume we could only scan newly added regions (in a second pass) to speed it up.